contracts/UFragmentsPolicy.sol

```diff
@@ -340,7 +371,27 @@ contract UFragmentsPolicy is Ownable {
             rebaseFunctionUpperPercentage,
             rebaseFunctionGrowth
         );
-        return uFrags.totalSupply().toInt256Safe().mul(rebasePercentage).div(ONE);
+
+        int256 currentSupply = uFrags.totalSupply().toInt256Safe(); // (or) supplyHistory[epoch]
```

Comment:
```suggestion
        int256 currentSupply = uFrags.totalSupply().toInt256Safe();
```
I don't think the supplyHistory has been written to yet, in the core rebase flow.